Twig/Extension/OAuthExtension.php

```diff
@@ -25,7 +25,7 @@ final class OAuthExtension extends AbstractExtension
     /**
      * @var OAuthHelper
      */
-    protected $helper;
+    private $helper;
 
     /**
      * @param OAuthHelper $helper
```

Comment:
You could also remove the constructor doc, it's a bit useless without proper description.

Reply:
I've seen some code style rules that can remove useless docblocks automatically. Maybe we should look into that. (and I can't wait to drop 7.3 support so we can use typed properties ;)).

Reply:
> I've seen some code style rules that can remove useless docblocks automatically. Maybe we should look into that.

Yes, afaik Symfony does the same.

Reply:
`no_superfluous_phpdoc_tags` is set to false. Should we change it?

Reply:
Yes, IMO we should change it. See https://github.com/hwi/HWIOAuthBundle/pull/1734 :)